openfisca_france/model/caracteristiques_socio_demographiques/capacite_travail.py

```diff
@@ -14,9 +14,10 @@ class taux_capacite_travail(Variable):
 class taux_incapacite(Variable):
     value_type = float
     entity = Individu
-    label = "Taux d'incapacité"
+    label = "Taux d'incapacité pris en compte pour l'AAH"
     definition_period = MONTH
     reference = 'https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=BD54F4B28313142C87FC8B96013E0441.tplgfr44s_1?idArticle=LEGIARTI000023097719&cidTexte=LEGITEXT000006073189&dateTexte=20190312'
-    documentation = "Taux d'incapacité retenu pour l'Allocation Adulte Handicapé (AAH)."
+    documentation = "Taux d'incapacité pris en compte pour l'allocation adulte handicapé (AAH)."
     is_period_size_independent = True
     set_input = set_input_dispatch_by_period
+    unit == '/1'
```

Comment:
```suggestion
    unit = '/1'
```

Mais je ne suis pas sûr que l'on peut mettre des unités pour les variables (comme on peut le faire pour les variabels).